message NoticeResponse{
	required string notice_id = 1; //表notes_config
    optional string player_name = 2; //玩家名
    optional string hero_no = 3; //武将no
    optional string hero_break_level = 4; // 武将突破等级
    optional string equipment_no = 5; //装备no
}
